src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -912,7 +913,7 @@ public async Task TestQueryFilterFieldAuth_UnauthorizedField()
                     }
                 }
             }";
-            string clientRoleHeader = AuthorizationType.Anonymous.ToString();
+            string clientRoleHeader = "limited-read-role";
```

Comment:
by changing the role from anonymous to `limite-read-role` you're changing the behavior of the test. The test description looks to be describing this test as validating behavior for the `anonymous` role.

Reply:
This test validate if include/exclude works or not. I am not seeing any specific scenario for `anonymous` here. I think description is misleading.